docs/2021-paper-reproduction.md

```diff
@@ -0,0 +1,47 @@
+# Reproducing results from the Guillaumin (2021) paper
+See README.md at repo root for further details. Here, we will provide just
+commands and commentary.
+
+## 1. Training data generation
+```
+python src/gz21_ocean_momentum/cli/data.py \
+--config-file examples/cli-configs/data-paper.yaml \
+--out-dir $(mktemp -d)
+```
+
+Unclear whether you may need `--ntimes 4000`.
+
+Configure `--out-dir` as you like.
+
+## 2. Model training
+WIP.
```

Comment:
Is this still WIP, or can it be used, potentially with documentation of what might not yet be ideal?

Reply:
I've added more instructions and training hyperparams directly from the paper (Table A1), and noted that I haven't tested it because training is borked for me. I think it's useful, since it gives you a pointer to running training with preconfigured hyperparams, spatial subdomains etc.

Reply:
Maybe give some more info then what you mean with WIP